--- a/valhalla/baldr/nodeinfo.h
+++ b/valhalla/baldr/nodeinfo.h
@@ -220,7 +220,8 @@
   bool can_contract() const {
     return edge_count() >= 2 && intersection() != IntersectionType::kFork &&
            type() != NodeType::kGate && type() != NodeType::kTollBooth &&
-           type() != NodeType::kTollGantry && type() != NodeType::kSumpBuster;
+           type() != NodeType::kTollGantry && type() != NodeType::kSumpBuster &&
+           type() != NodeType::kBorderControl;
   }
 
   /**
